pkg/http/api/v1/handler.go

```diff
@@ -186,6 +191,13 @@ func (h *requestHandler) GetScanReport(res http.ResponseWriter, req *http.Reques
 }
 
 func (h *requestHandler) GetMetadata(res http.ResponseWriter, req *http.Request) {
+	vi, err := h.wrapper.GetVersion()
+	if err != nil {
+		log.WithError(err).Error("Error while retrieving vulnerability DB version")
+		h.SendInternalServerError(res)
```

Comment:
According to the pluggable scanners spec we're supposed to return an `harbor.Error` stuct: https://github.com/goharbor/pluggable-scanner-spec/blob/master/api/spec/scanner-adapter-openapi-v1.0.yaml#L60

```suggestion
		h.WriteJSONError(res, harbor.Error{
			HTTPCode: http.StatusInternalServerError,
			Message:  "cannot retrieve vulnerability DB version",
		})
```

Reply:
Nice, I didn't know about it. Fixed here https://github.com/aquasecurity/harbor-scanner-trivy/pull/87/commits/ede8e8e1b1de62f83734bc9b4e1375c1b6e62163